source/sas_compress.cpp

```diff
@@ -69,13 +70,15 @@ class ZlibCompressor : public SAS::Compressor
   char _buffer[4096];
 };
 
+typedef std::pair<LZ4_stream_t*, struct preserved_hash_table_entry*> saved_lz4_stream ;
```

Comment:
Would prefer the typedef to either be camel-cased or ent with _t. At the moment it looks like a variable name.